qa/acceptance/spec/lib/cli/logstash-plugin/install_spec.rb

```diff
@@ -0,0 +1,74 @@
+# encoding: utf-8
+require_relative "../../../spec_helper"
+require "logstash/version"
+require "fileutils"
+require "logstash/devutils/rake"
```

Comment:
we should move most of the code within https://github.com/elastic/logstash-devutils/blob/master/lib/logstash/devutils/rake/vendor.rake  to a real class that can be propery reused, there is also some duplicated code in our logstash-core rake tasks.